### PR TITLE
Cleanup & Add contribution guide

### PR DESCRIPTION
**Pull request by ariG23498**
This PR add a contribution guide, and sorts the notebooks into folder for easy navigation.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Welcome to the contribution guide 🤗
+
+We are excited to invite the community to contribute to the repository! We appreciate all contributions, big or small. Your efforts help make this repository a valuable resource for everyone working with Llama 3.x models in the Hugging Face ecosystem.
```
Comment (osanseviero) [inline .github/CONTRIBUTING.md:3]:
```suggestion
We are excited to invite the community to contribute to the repository! We appreciate all contributions, big or small. Your efforts help make this repository a valuable resource for everyone working with Llama models.
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - Navigate to the original repository and click on "New Pull Request"
+    - Compare across forks and select your branch.
+    
+    ![pull request](../assets/PR.png)
+    - Provide a clear description of your contribution.
+   
+
+## 💡 Need Help?
+
+If you have any questions or need guidance, feel free to open an issue or start a discussion. We're here to help!
```
Comment (osanseviero) [inline .github/CONTRIBUTING.md:45]: Let's concentrate discussions on issues/PRs. I think we should not enable discussions for this repo as it's too small at the moment.

```suggestion
If you have any questions or need guidance, feel free to open an issue or draft PR. We're here to help!
```

**Review by osanseviero — APPROVED** [review]
LGTM!

**Comment by ariG23498** [comment]
Merging and opening issues for contributions!

**Pull request merged**